warehouse/query-core/src/main/java/datawave/query/config/LookupUUIDTune.java

```diff
@@ -125,6 +125,8 @@ public void configure(GenericQueryConfiguration configuration) {
             if (maxShardsPerDayThreshold != -1) {
                 rsqc.setShardsPerDayThreshold(maxShardsPerDayThreshold);
             }
+            rsqc.setEnforceUniqueTermsWithinExpressions(enforceUniqueTermsWithinExpressions);
+            
```

Comment:
we only need to set this once. If the value is stored in the BaseQueryLogic, no need to set it again in the ShardQueryConfiguration

Reply:
I'll make that change, thanks